lib/src/screens/connect_hotspot_prefix.dart/widgets/connect_hotspot_prefix_screen.dart

```diff
@@ -95,8 +95,12 @@ class _ConnectHotspotPrefixScreenState extends State<ConnectHotspotPrefixScreen>
                               children: [
                                 Icon(Icons.check_circle, color: Colors.green),
                                 const SizedBox(width: 8.0),
-                                Text("You are connected to the device's hotspot.", style: listStyle),
-                                Spacer(),
+                                Expanded(
+                                  child: Text(
+                                    "You are connected to the device's hotspot.",
+                                    style: listStyle,
+                                  ),
+                                ),
```

Comment:
<img width="400" height="866" alt="IMG_0869" src="https://github.com/user-attachments/assets/e8caf4de-d382-4ad2-8abb-ca779b49dbe1" />
not cutoff anymore